pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
+| **Step 4**  | ❎           | **Special devices (CD, DVD etc..)**      | **Unplug all special devices.**                            | - Nothing must be plugged into to migrated vSphere managed data centers. All special devices (CD, DVD etc..) must be removed before migration, otherwise they will be force-removed by the migration process and lost. |                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
+| **Step 5**  | ❎           | **Datastore  clusters**                  | **Delete all clustering rules.**                           | - Clustering rules will have to be deleted before migration because this notion do not exist on the VCD side.                                                                                                          |                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
+| **Step 6**  | ❎           | **Memory over-commitment**               | **Scale up VCD.**                                          | - Foresee more resources on the VCD side or resize/optimize the VMs before migration on the vSphere on OVHcloud side. <br/> Because you won't be able to over-commit resources.                                        | [Modifying virtual machine resources](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/modify_hardware_configuration_of_vm)                                   |                                                                                                                                                
```

Comment:
"Foresee more resources on the VCD side or resize/optimize the VMs before migration on the vSphere on OVHcloud side." Not sure about this sentence. We should rephrase it, the end is hard to read.